### PR TITLE
Update deathMessages.json

### PR DESCRIPTION
**Pull request by Tman2143**
death mesaged

### CODE REVIEW

File: src/me/bobthe28th/capturethefart/ctf/deathMessages.json [hunk 1]
```diff
@@ -2,12 +2,16 @@
...
     "default": ["$1 died","$1 farted","/killed $1"],
     "wizardZap": ["$1 got zapped"],
-    "FALL": ["$1 fell and didn't get up","$1 went splat","$1 falled","$1 is malding"]
+    "FALL": ["$1 fell and didn't get up","$1 went splat","$1 falled","$1 is malding","$1 fucking fell over","$1 "]
   },
   "deathbe": {
     "default": ["$1 was killed by $2","$1 could not handle the awesomeness of $2","$1 got fuccking died by $2"],
-    "wizardZap": ["$1 got zapped by $2"],
-    "demoTNT": ["$1 got blown into a million pieces by $2"],
+    "wizardZap": ["$1 got zapped by $2","$1 got electrocuted to fucking death by $2","$1 got absolutally smited to death by $2"],
```
Comment (Bobthe28th) [inline src/me/bobthe28th/capturethefart/ctf/deathMessages.json:9]:
```suggestion
    "wizardZap": ["$1 got zapped by $2","$1 got electrocuted to fucking death by $2","$1 got absolutely smited to death by $2"],

```

Comment (Bobthe28th) [inline src/me/bobthe28th/capturethefart/ctf/deathMessages.json:9]:
```suggestion
    "wizardZap": ["$1 got zapped by $2","$1 got electrocuted to fucking death by $2","$1 got absolutally smote to death by $2"],

```

File: src/me/bobthe28th/capturethefart/ctf/deathMessages.json [hunk 2]
```diff
@@ -2,12 +2,16 @@
...
+    "wizardFire": ["$1 was incinerated by $2","$1 got fucking scorched by $2","$1 burnt to a crisp due to $2"],
+    "wizardFreeze": ["$1 frozed to death by $2","$1 god frostbitted by $2","$1 hit absolute zero from $2"],
+    "builderAxe": ["$1 got hella hella 90s built on them by $2","$1 lost the build battle to $2","$1 could not escape maze built by $2"],
```
Comment (Bobthe28th) [inline src/me/bobthe28th/capturethefart/ctf/deathMessages.json:12]:
```suggestion
    "builderAxe": ["$1 got hella 90s built on them by $2","$1 lost the build battle to $2","$1 could not escape maze built by $2"],

```

**Review by Bobthe28th — CHANGES_REQUESTED** [review]
You used "fucking" in like half the messages

File: src/me/bobthe28th/capturethefart/ctf/deathMessages.json (hunk 1 above)
Comment (Bobthe28th) [reply src/me/bobthe28th/capturethefart/ctf/deathMessages.json:9]: Just spell 4head

File: src/me/bobthe28th/capturethefart/ctf/deathMessages.json (hunk 2 above)
Comment (Bobthe28th) [reply src/me/bobthe28th/capturethefart/ctf/deathMessages.json:12]: Prob when you fixed order of $1 and $1 you made extra hella?

**Comment by Bobthe28th** [comment]
Also there is wizardZap (left click) and wizardLightning (right click) idk if that will change the death messages but yea

**Comment by Bobthe28th** [comment]
Also this is #69, nice

**Comment by Tman2143** [comment]
> You used "fucking" in like half the messages

yea that's like half the fun

**Comment by Bobthe28th** [comment]
can you look at the changes I requested?

File: src/me/bobthe28th/capturethefart/ctf/deathMessages.json (hunk 1 above)
Comment (Bobthe28th) [reply src/me/bobthe28th/capturethefart/ctf/deathMessages.json]: Ill resolve

**Review by Bobthe28th — APPROVED** [review]
Zamn!

**Comment by Bobthe28th** [comment]
> Also there is wizardZap (left click) and wizardLightning (right click) idk if that will change the death messages but yea

also this

**Pull request merged**